home/src/main/resources/rdf/tbox/filegraph/vitro-dynamic-api.owl

```diff
@@ -399,6 +399,8 @@
     <!-- https://vivoweb.org/ontology/vitro-dynamic-api#action -->
 
     <owl:Class rdf:about="https://vivoweb.org/ontology/vitro-dynamic-api#action">
+        <rdfs:subClassOf rdf:resource="https://vivoweb.org/ontology/vitro-dynamic-api#operation"/>
+		<rdfs:subClassOf rdf:resource="https://vivoweb.org/ontology/vitro-dynamic-api#parameterProvider"/>
```

Comment:
Please correct indentation

Reply:
Not sure Action shouldn't be parameterReceiver as well. Think also about the case when we are chaining an action in the another action which we discussed once.

Reply:
I found later in your code, there are requiredParams for actions but it is calculated based on set of steps, not defined in RDF. Just ignore above comment.

Reply:
Indentation is fixed